src/utils/vc/custom_crypto.js

```diff
@@ -35,12 +38,15 @@ export {
   EcdsaSecp256k1SigName,
   Ed25519VerKeyName,
   Ed25519SigName,
+  Ed255192020VerKeyName,
+  Ed255192020SigName,
   Sr25519VerKeyName,
   Sr25519SigName,
   EcdsaSecp256k1VerificationKey2019,
   EcdsaSecp256k1Signature2019,
   Ed25519VerificationKey2018,
   Ed25519Signature2018,
+  Ed25519Signature2020,
```

Comment:
`/contexts/security_context` needs update?

Reply:
no, this key has its own context. wont work outside of our SDK if we put 2020 context in there